src/gui_tool/gui/general_gui.py

```diff
@@ -141,6 +143,16 @@ def play_video(self):
                     self.get_mode_handler().INSTRUCTIONS,
                 )
                 window.run()
+            elif self.key_mapper.consume("j"):
+                loc = TextPopup("Enter Index").run()
+                try:
+                    iloc = int(loc)
+                    if iloc >= 0 and iloc <= self.vcm.get_frames_count() - 1:
+                        self.vcm.start_from(iloc)
```

Comment:
For the same reason, can you add "self.logger.log("Jumped to index {0}".format(iloc))" if you think this should be merged in